jasmine-standalone-2.5.2/spec/invertedIndex.spec.js

```diff
@@ -0,0 +1,211 @@
+const values = require('object.values');
+
+//  import test books
+const book = require('../../json files/books.json');
+const emptyJsonFile = require('../../json files/emptyJsonFile.json');
+const emptyArrayFile = require('../../json files/emptyArrayFile.json');
+const invalid = require('../../json files/invalidFile.json');
```

Comment:
I think 'invalid' is not descriptive enough as a variable name, as invalid can mean different thing